core/primitives/strobe.hpp

```diff
@@ -203,6 +207,15 @@ namespace kagome::primitives {
       beginOp<kMore, kFlag_A | kFlag_C>();
       overwrite(data);
     }
+
+    auto data() {
+      return gsl::make_span(as<const uint8_t>(), count<uint8_t>() + 3ull);
+    }
+
+    auto state() const {
+      return std::make_tuple(
```

Comment:
I believe it would be more convenient if you return a struct with named parameters, like StrobeState, instead of a tuple.